complete-application/mysite/app/templates/account.html

```diff
@@ -11,9 +11,9 @@
         <img src="https://fusionauth.io/cdn/samplethemes/changebank/changebank.svg" />
         <div class="h-row">
           <p class="header-email">{{ email }}</p>
-          <form id="logoutForm" class="button-lg"  action="{% url 'oidc_logout' %}" method="post">
+          <form action="{% url 'oidc_logout' %}" method="post">
             {% csrf_token %}
-            <a class="button-lg" href="#" onclick="document.getElementById('logoutForm').submit();">Logout</a>
+            <input type="submit" value="Logout" class="button-lg">
```

Comment:
@hollygirouard , please help with the style for the submit button. It should match the Login anchor.

Reply:
@mark-robustelli Take a peep at this locally and let me know if you have any feedback. I just pushed it up. 
![Screenshot 2024-06-18 at 10 58 40 AM](https://github.com/FusionAuth/fusionauth-quickstart-python-django-web/assets/20582692/33a309a8-ec6a-401c-87c9-e128d909b163)

Reply:
@hollygirouard Have you tried Chrome.  Still looks off.  
![Screenshot 2024-06-18 at 12 47 20 PM](https://github.com/FusionAuth/fusionauth-quickstart-python-django-web/assets/137117976/a663dde0-d671-40b1-af92-c4c11873c06d)

Reply:
<img width="865" alt="Screenshot 2024-06-18 at 1 56 25 PM" src="https://github.com/FusionAuth/fusionauth-quickstart-python-django-web/assets/20582692/40916b64-2f18-497e-bc3a-d67a21ef5f19">

Yes, try command + shift + R. Django likes to cache pretty hard.

Reply:
Woop woop! You're very welcome!